front-end/src/components/DashBoardLayout/UserPopover/UserPopover.tsx

```diff
@@ -74,6 +95,9 @@ const UserPopover = () => {
           <ListItemText>Logout</ListItemText>
         </MenuItem>
       </Menu>
+
+      {showProfile && <Profile onClose={handleCloseProfile} />}
```

Comment:
This component I made is a popover like click and dropdown menu with three options. 
The above two should not be called here. They should be two new pages with corresponding /views and page.tsx created like. /app(dashboard)/settings/profile/page.tsx. Clicking on the option will navigate to the page, like how github right top is doing.